components/layout/ModernHeader.tsx

```
@@ -0,0 +1,116 @@
+              <div role="menu" aria-label="Select network" className="absolute right-0 mt-2 w-44 rounded-lg glass-panel p-1.5 z-50 border border-[var(--glass-border-strong)] shadow-lg">
+                <ul className="flex flex-col gap-0.5">
+                  {ENVIRONMENTS.filter(e => ['devnet','mainnet-beta'].includes(e.label)).map(env => {
+                    const active = env.label === ctx.environment.label
+                    return (
+                      <li key={env.label}>
+                        <button
+                          role="menuitemradio"
+                          aria-checked={active}
+                          onClick={() => setCluster(env.label as Cluster)}
+                          className={cn("w-full text-left px-3 py-2 rounded-md text-sm flex items-center gap-2 transition-colors", active ? 'bg-[var(--glass-bg-alt)] text-white' : 'text-gray-300 hover:bg-[var(--glass-bg-alt)]')}
+                        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)